CHANGELOG.md

```diff
@@ -3,6 +3,96 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.6
+
+_Dec 22, 2023_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Data Grid now supports `Date` objects in the `filterModel`
+- 🌍 Improve Russian (ru-RU) locale on the Data Grid
+- 🐞 Bugfixes
+
+### Data Grid
+
+#### Breaking changes
+
+- The filter panel no longer uses the native version of the [`Select`](https://mui.com/material-ui/react-select/) component for all components.
+- The `getOptionValue` and `getOptionLabel` props were removed from the following components:
+
+  - `GridEditSingleSelectCell`
+  - `GridFilterInputSingleSelect`
+  - `GridFilterInputMultipleSingleSelect`
+
+  Use the `getOptionValue` and `getOptionLabel` properties on the `singleSelect` column definition instead:
+
+  ```tsx
+  const column: GridColDef = {
+    type: 'singleSelect',
+    field: 'country',
+    valueOptions: [
+      { code: 'BR', name: 'Brazil' },
+      { code: 'FR', name: 'France' },
+    ],
+    getOptionValue: (value: any) => value.code,
+    getOptionLabel: (value: any) => value.name,
+  };
+  ```
+- The `filterModel` now supports `Date` objects as values for `date` and `dateTime` column types.
+  The `filterModel` still accepts strings as values for `date` and `dateTime` column types,
+  but all updates to the `filterModel` coming from the UI (e.g. filter panel) will set the value as a `Date` object.
+
+#### `@mui/x-data-grid@7.0.0-alpha.6`
+
+- [DataGrid] Fix typos in the JSDoc (#11451) @flaviendelangle
+- [DataGrid] Make `checkboxSelection` respect the `disableMultipleRowSelection` prop (#11448) @cherniavskii
+- [DataGrid] Support `Date` objects in filter model (#7069) @cherniavskii
+- [DataGrid] Use non-native `Select`s by default (#11330) @cherniavskii
+- [l10n] Improve Russian (ru-RU) locale (#11441) @wensiet
+
+#### `@mui/x-data-grid-pro@7.0.0-alpha.6` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@7.0.0-alpha.6`.
+
+#### `@mui/x-data-grid-premium@7.0.0-alpha.6` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.0.0-alpha.6`.
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@7.0.0-alpha.6`
+
+- [fields] Adjust `PickersInput` sizing styles (#11392) @noraleonte
+- [fields] Fix section pasting (#11447) @LukasTy
+- [pickers] Add `PickersTextField` `standard` and `filled` variants (#11250) @noraleonte
+- [pickers] Add missing breaking changes to changelog (#11420) @MBilalShafi
```

Comment:
IMHO, this has a bit incorrect prefix. It only impacts the `changelog` file, not the package. 🙈 
```suggestion
```

Reply:
Missed these, I'll do a patch later.